README.rst

```diff
@@ -30,13 +30,286 @@
+
+  * --api-version, --http-health-port, --grpc-infer-thread-count,
+    --grpc-stream-infer-thread-count,--allow-poll-model-repository
+    and --allow-model-control are removed.
+
+  * The default for --model-control-mode is changed to *none*.
+
+* The HTTP/REST and GRPC protocols, while conceptually similar to
+  version 1, are completely changed in version 2. See the `inference
+  protocols
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/http_grpc_api.html>`_
+  section of the documentation for more information.
+
+* Python and C++ client libraries are re-implemented to match the new
+  HTTP/REST and GRPC protocols. The Python client no longer depends on
+  a C++ shared library and so should be usable on any platform that
+  supports Python. See the `client libraries
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/client_library.html>`_
+  section of the documentaion for more information.
+
+* The version 2 cmake build requires these changes:
+
+  * The cmake flag names have changed from having a TRTIS prefix to
+    having a TRITON prefix. For example, TRITON_ENABLE_TENSORRT.
+
+  * The build targets are *server*, *client* and *custom-backend* to
+    build the server, client libraries and examples, and custom
+    backend SDK, respectively.
+
+* In the Docker containers the environment variables indicating the
+  Triton version have changed to have a TRITON prefix, for example,
+  TRITON_SERVER_VERSION.
+
+Roadmap
+-------
+
+The 20.03.1 release of Triton consists of a single server/container
+that supports both the existing version 1 APIs and protocols and the
+new version 2 APIs and protocols. For version 2 the release is beta
+quality and includes the new `HTTP/REST and GRPC protocols
+<https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2>`_
+and corresponding new C++ and Python client libraries. Version 2 also
+includes a beta release of the new server C API defined in
+tritionserver.h.
+
+The upcoming 20.06 release of Triton will include two separate server
+containers:
+
+* A legacy V1 version of Triton will be released from the master-v1
+  branch. The NGC container for the V1 version of Triton will be
+  called tritonserver:20.06-v1-py3. The V1 version of Triton is
+  deprecated and no releases beyond 20.06 are planned. The V1 version
+  of Triton maintains backwards compatibility with prior V1 versions
+  in both the server APIs and in the C++ and Python libraries. See the
+  `master-v1 branch README
+  <https://github.com/NVIDIA/triton-inference-server/tree/master-v1>`_
+  for more information.
+
+* The new V2 version of Triton will be released from the master branch
+  and will include the new GRPC and HTTP protocols based on `inference
+  protocols
+  <https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2>`_
+  that have been proposed by the `KFServing project
+  <https://github.com/kubeflow/kfserving>`_. Version 2 of Triton will
```

Comment:
"... Triton has a new C API and new ...."

".. NGC container... is called. ..."